hs_geographic_feature_resource/templates/pages/geographicfeatureresource.html

```diff
@@ -0,0 +1,235 @@
+{#                        {% endif %}#}
+{#                    {% endif %}#}
+{#                {% endfor %}#}
+{##}
+{#            </div>#}
+{#        {% endif %}#}
+
+
+    {% endblock %} {# extended block ends here #}
+
+
+{%  endblock %}
+
+
+{% block extra_js %}
+
+{{ block.super }}
+
+{# put resource specific js below #}
+<script type="text/javascript">
+    // original coverage meta check
+    function check_ori_meta_status(){
+        var form_action = $("form[id='id-originalcoverage']").attr('action');
+        if (form_action.indexOf('update-metadata') > -1) {
+            var new_action = form_action.replace('update-metadata', 'delete-metadata');
+            var modal_body_text = '<strong>Are you sure you want to delete this metadata element?</strong>'
+            $("#delete-original-coverage-element-dialog").find(".modal-body").html(modal_body_text);
+            $("#delete-original-coverage-element-dialog").find(".modal-footer").children("a").attr('href',new_action);
+            $("#delete-original-coverage-element-dialog").find(".modal-footer").children("a").show()
+        }
+        else {
+            var modal_body_text = '<strong>There is no original coverage metadata for this resource to delete.</strong>'
+            $("#delete-original-coverage-element-dialog").find(".modal-body").html(modal_body_text);
+            $("#delete-original-coverage-element-dialog").find(".modal-footer").children("a").hide()
+        }
+    };
+
+    del_msg_shp_shx_dbf = "<font color='red'>BE CAREFUL!</font> The 'shp', 'shx' and 'dbf' are three mandatory files. Missing of any of them will corrupt data integrity. So this operation is about to permanently <font color='red'>DELETE ALL EXISTING FILES</font> of this resource! You may upload new shapefiles later."
+    del_msg_prj = "This operation is about to remove coordinate system metadata!"
+    $(document).ready(function(){
+
+         $('.label-file-name').each( //iterate
+                 function()
+                 {
+                     filename=$(this).text()
+                     extension=filename.substring(filename.length-4, filename.length)
+                     if (extension == ".shp" || extension == ".shx" || extension == ".dbf")
+                     {
+                         tag_id=$(this).parent().parent().next().children().find("span[title='Delete']").attr('data-target')
+                         tag_id2= tag_id.substring(1, tag_id.length)
+                         $("div[id='"+tag_id2+"']").children().find("div[class='modal-body']").html(del_msg_shp_shx_dbf)
+                     }//if
+                     else if (extension == ".prj")
+                     {
+                         tag_id=$(this).parent().parent().next().children().find("span[title='Delete']").attr('data-target')
+                         tag_id2= tag_id.substring(1, tag_id.length)
+                         $("div[id='"+tag_id2+"']").children().find("div[class='modal-body']").html(del_msg_prj)
```

Comment:
@ironfroggy I changed this line to:
$(' "# ' + tag_id2 + ' " ').children().find("div[class='modal-body']").html(del_msg_prj)
but it is not working, because tag_id2  is a variable?